apps/console/src/styles/app.css

```
@@ -1233,3 +1233,325 @@ table.records-table tbody tr.active td { background: transparent; }
+  display: flex;
+  gap: 6px;
+  flex-wrap: wrap;
+}
+
+.kg-detail-title {
+  font-family: 'Lora', Georgia, serif;
+  font-size: 1.125rem;
+  font-weight: 500;
+  color: var(--app-fg);
+  line-height: 1.3;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)